app/packages/operators/src/types.ts

```diff
@@ -414,6 +434,45 @@ export class File extends OperatorObject {
       description: "The contents of the directory",
     });
   }
+
+  static fromJSON(json: any): File {
+    return new File(OperatorObject.propertiesFromJSON(json));
+  }
+}
+
+/**
+ * Operator type for defining an uploaded file and its metadata.
+ */
+
+export class UploadedFile extends OperatorObject {
+  constructor(public properties: Map<string, Property> = new Map()) {
+    super(properties);
+    this.str("name", {
+      label: "Name",
+      description: "The name of the uploaded file",
+    });
+    this.str("type", {
+      label: "Type",
+      description: "The mime type of the uploaded file",
+    });
+    this.int("size", {
+      label: "Size",
+      description: "The size of the uploaded file in bytes",
+    });
+    this.str("contents", {
```

Comment:
Can you change this to `content` see https://github.com/voxel51/fiftyone/pull/3699